src/lib/common/HighlightManager.ts

```diff
@@ -211,6 +211,11 @@ export class HighlightManager {
 	public async notify(message: BushMessage, user: Snowflake, hl: HighlightWord): Promise<boolean> {
 		assert(message.inGuild());
 
+		const userObject = await message?.guild?.members?.cache?.get(user);
```

Comment:
why would you block highlight while online, people arent always monitoring every guilds general. this should at least be a toggleable option